### ISSUE TITLE
Calligra Icons look like mimetype

### ISSUE THREAD

**Issue by staniek** [NitruxSA/plasma-next-icons#70]
Breeze design language defines offers colours for app icons to set them apart from action/mimetype icons. Calligra icons look like mimetype ones, and more - close to generic office icons. Is this temporary?

Example for Kexi (compare with other apps out there): 

![](http://i.imgur.com/X0L3agt.png)

PS: Kexi will redesign logo and icon in 3.0 so I don't want waste your time before this happens. For 2.9 (to be released in 2 weeks) I can set current Kexi icon forcibly http://commons.wikimedia.org/wiki/File:Calligra_Kexi_icon.svg in the app or you can adapt it to the taste of Breeze, then new icon can be delivered for 3.0.

**Comment by DarkknightAK** [comment]
The idea for the calligra office suite was to have the same design language for the office suite and to be close to Libre Office because the people knows LO as a office suite.

But I have to say you are right. The icons are very close to the mimetype icons.

An additional problem for me (I made this icons) was that only the webpage has an cooporate design. Maybe you can help my to find an better icon. For the background I want to use something that fitts the Calligra suite and in front I want to use individual uniquie labels.

**Comment by staniek** [comment]
Hi! The approach of unifying apps (LO Calc-Calligra Sheets, etc.) isn't a good idea in practice. Apps have their personalities and nonzero population uses some from both suites. The challenge is not to alter branding.

After all, we'll change icons on docs and wikipedia and so on pages to reflect breeze icons. If these are mostly the same as in LO there's a risk of confusion.

http://commons.wikimedia.org/wiki/File:Calligra_Kexi_icon.svg can be quite easily Breeeze-ified. To show what I mean, note that I like how the Inkscape icon was reworked, it's still recognizable.
I also like how the firefox icon was altered but my kids can still say what it is.

Summing up, other calligra apps than Kexi are probably not going to change their icons (will double check by asking though) so they might be available for rework already, currently you can find them  named hisc-app-calligra*.svgz in the calligra git repo. (preview of them are at http://en.wikipedia.org/wiki/Calligra_Suite#Components). 

Probably changes can include altered colours, no idea if the slight rotation must be removed or not.

**Comment by DarkknightAK** [comment]
I will remove the app icons for the calligra suite and will look if I can find a better solution. As you writen we have some time until 3.0 will be released.

**Comment by DarkknightAK** [comment]
The problem is that some app icons (like sheets, stage, word) are not realy app icons they show more what the app will do.
Maybe we can use the circle (like the Karbon and Krita icon) to have an suite design and in the circle the icon. Or do you think that each app should have there unique icon.

**Comment by staniek** [comment]
> I will remove the app icons for the calligra suite and will look if I can find a better solution. As you writen we have some time until 3.0 will be released.

Thanks, this looks as a simplest solution; I won't need to alter any code in Kexi :)

> The problem is that some app icons (like sheets, stage, word) are not realy app icons they show more what the app will do.

I'm all for loosening this relation, to have icons more like firefox or inkscape or chrome and here's a challenge for artists/designers. One talented designer works on Kexi branding now and I can only say the relation to the function would disappear.

> Or do you think that each app should have there unique icon.

I'll contact Calligra app maintainers. I think the times of "suites" are over as much as times of boxed software. Calligra apps are tied together based on common technology (a good factor) but everyone is free to install individual apps. Mobile/desktop app stores changed the users perception too. Also boundaries between apps would and should blur over time, I think, while the level of integration increases: people want to read/write rich text in their email app directly see the tables, etc.

Emphasizing the fact that there's a suite of apps right in the icon design does not seem to deliver knowledge our users need or want. MS goes this way [http://en.wikipedia.org/wiki/Microsoft_Office_2013#mediaviewer/File:Microsoft-Office-2013-lineup.svg] but I am afraid if we want to do that too without playing catch-up. The rounded rectangle's corner of the current Breeze calligra icons suggest similarities to MS Office 2007's approach [http://i1-win.softpedia-static.com/screenshots/MS-Office-2007-Icons-Pack_1.png].

On your taskbar there are icons of browser(s), some editor, painting app, etc. Relationship between them aren't important, the more different they are each other (at 32x32 size?), the better. I believe that normal user sees Kate and Words to be closer together than Kexi and Words for example. But in the same time I agree Kate and Words should not look similar.
